_posts/2021-02-01-aws-config-compliance-automation.md

```diff
@@ -0,0 +1,95 @@
+
+While AWS Config does allow for remediation to be automated through a combination with AWS Systems Manager Automation documents, automated remediation is not always a desirable outcome. Often, automated remediation of compliance violations can cause application outages if not handled correctly, and the outage impact can be larger than the impact of allowing the non-compliance to extend for a time while appropriate manual remediation occurs.
+
+This is not the case in every scenario, but decisions need to be made about either automating some remediation actions or none. If none, remediation still needs to occur, and normally the compliance department will have specified the timeframe for which a non-compliant asset needs to be remediated or face further consequences.
+
+## ServiceNow
+
+In large companies, many choose to utilize workflow automation software, and one of the most popular of these systems, is the cloud-based <a href="https://www.servicenow.com/">ServiceNow</a> platform. ServiceNow can be utilized in a variety of ways, but normally it is used as the source-of-truth for internal configuration management database (CMDB) needs, allowing an organization to have visibility and centralized control of disperse assets. Another use of ServiceNow is to help manage IT processes around internal Help Desk, as well as problem and incident tracking. These types of requests generally blend together with CMDB needs to the point where a central tool encompassing all of these needs is very beneficial.
+
+When considering the normal IT process workflows that some companies help to automate with the help of ServiceNow, compliance needs become very similar. The normal mode of operation for IT assets would be to have them in compliance with regulations and policy, so when an asset is found to be non-compliant, this is an issue just like several other IT issues (service outages, request tracking, other incicents, etc), and as such it makes sense to track utilizing the same system as those other issues.
+
+If ServiceNow is accurately reflecting configuration item (CI) data in the CMDB, there are some useful direct integrations possible using a combination of AWS Config, AWS Simple Notification Service (SNS), and ServiceNow. However, many organizations do not have a 1:1 reflection of AWS resources as CI's in their ServiceNow CMDB, so some customization is required. Unfortunately when this is necessary, the <a href="https://docs.servicenow.com/bundle/paris-it-operations-management/page/product/cloud-management-v2-setup/task/aws-config-service-cloud-mgt.html">out-of-the-box capabilities utilizing the tools</a> is not possible.
```

Comment:
The last sentence feels a little awkward to me as well. Maybe, "Unfortunately, when customization is necessary, these out-of-the-box capabilities are not possible."